tests/integration/records/records_numeric_database_fixture.py

```diff
@@ -50,6 +50,53 @@ def bring_up(self):
                      1234.56::NUMERIC(6, 2) AS fixed_6_2,
                      12147483647.78::REAL AS float32,
                      19223372036854775807.78::FLOAT8 AS float64;
+"""  # noqa
+        elif self.engine.name == 'mysql':
+            # Postgres supports a number of different numeric types
+            # https://dev.mysql.com/doc/refman/8.0/en/numeric-types.html
+            #
```

Comment:
Uncompleted punchlist item detected--consider resolving or moving this to your issue tracker